js/components/file.js

```diff
@@ -29,29 +29,28 @@ Fliplet.FormBuilder.field('file', {
     mediaFolderNavStack: {
       type: Array,
       default: []
+    },
+    value: {
+      type: Array,
+      default: []
     }
   },
   computed: {
     selectedFileName: function() {
-      var names = [];
-
-      for (var i = 0; i < this.selectedFiles.length; i++) {
-        var file = this.selectedFiles.item(i);
-        names.push(file.name);
-      }
-
-      return names.join(', ');
+      return _.map(this.selectedFiles, 'name').join(', ');
     }
   },
   methods: {
     updateValue: function() {
       var $vm = this;
-      this.selectedFiles = this.$refs.fileInput.files;
+
+      this.selectedFiles.splice(0, this.selectedFiles.length);
```

Comment:
What's the logic and necessity of this particular part? Emptying the array by splicing seems like an odd method 🤔

Reply:
I see. Fine with me 🆗